rpc/README.md

```diff
@@ -98,7 +98,6 @@ The crate `ckb-rpc`'s minimum supported rustc version is 1.71.1.
         * [Method `get_deployments_info`](#method-get_deployments_info)
     * [Module Subscription](#module-subscription)
         * [Method `subscribe`](#method-subscribe)
-        * [Method `unsubscribe`](#method-unsubscribe)
```

Comment:
Why remove the `unsubscribe` method?

Reply:
we still have `unsubscribe`, but it's specified with `subscribe` and generated by `jsonrpc-utils`:

```rust
  type S: Stream<Item = PublishMsg<String>> + Send + 'static;
    #[rpc(pub_sub(notify = "subscribe", unsubscribe = "unsubscribe"))]
    fn subscribe(&self, topic: Topic) -> Result<Self::S>;
```

I will think about how we generate the `RPC` API document.

Reply:
Fix in https://github.com/nervosnetwork/ckb/pull/4187